### DISCUSSION TITLE
Repurpose Speakers Bureau for new Contributors & Experts Program

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/discussions#16] [Community Working Group]
Proposal to repurpose the Speakers Bureau platform for a new Contributors & Experts Programme

- Current platform is not dynamic and is difficult to update
- Records are static and 'past talks' are not taken into account
- Limited filtering ability through search only

**Proposal**
Use existing platform with following changes:

- More attributes for filtering
- Ability to update records
- 1 year limit on records without update
- Ability to add contributions outside of speaking (OSS, Organising, Writing, Thought Leadership, Podcast participation etc)
- Ability for background scoring ability to add tier levels

**Why**
 - Broader catalogue of contributors
 - Up to date records
 - Linked to Experts programme which would recognise top contributors
 - Mid-Tier status level to incentivise participation

**Comment by tmcclell** [comment]
Include internal advocacy contributions. Sarah asked about students but Namrata says that we may not be ready for different criteria for students. If someone has bandwidth to work on this - please comment. Discussion about ratings - contributions, experts, top performers, etc? Timeline targets COP for delivery for first phase, January for second phase. CWG approved.

**Comment by adamj89** [comment]
Update on this project - now that this is approved by the CWG, @adamj89 will be delivering a 1-pager to @jawache for consideration. This will then go forward to scope out the technical work and timeline.

Our aspiration is for the new program to launch in time for Decarbonize Software.
